programs/vote/src/vote_state.rs

```diff
@@ -162,11 +162,11 @@ impl VoteState {
     ///  if commission calculation is 100% one way or other,
     ///   indicate with false for was_split
     pub fn commission_split(&self, on: f64) -> (f64, f64, bool) {
-        match self.commission {
+        match self.commission.min(100) {
```

Comment:
@ParthDesai an easy way to keep numbers larger than 100 from messing stuff up ;)

Reply:
@rob-solana I think this is not a good idea. This will mask actual bug, that was allowing `commission` to be greater than zero in first place. 

Rather, I believe we should have assertion here. And actual validation need to be there while we are assigning the value.

Reply:
I'm confused by "greater than zero"

Reply:
@rob-solana Sorry, I meant greater than 100.

Reply:
I think the min(100) can be removed now, if you like

Reply:
where was there a bug that was masked?

Reply:
@rob-solana Sorry, I was very tired, when I wrote that comment. I meant this scenario could happen, if we chose to normalize commission percent, kind of like throwing actual bug under rug (Maybe we are not reading proper value from CLI, etc etc.)

Reply:
no problem.  I went looking for all mentions of commision for this PR, but could easily have missed one